docs/Using-EthSigner/Using-EthSigner.md

```diff
@@ -0,0 +1,82 @@
+description: Using EthSigner
+<!--- END of page meta data -->
+
+# Using EthSigner
+
+EthSigner provides transaction signing and access to your keystore by implementing the following JSON-RPC 
+methods:   
+
+* [`eth_accounts`](#eth_accounts)
+
+* [`eth_sendTransaction`](#eth_sendtransaction)
+
+All other JSON-RPC requests are passed through EthSigner to Pantheon and the result from Pantheon returned 
+by EthSigner. 
+
+!!! note 
+    EthSigner supports the JSON-RPC service over HTTP only. 
```

Comment:
Do you mean only over HTTP and not HTTPS or only over HTTP/S but not Websocket ?

Reply:
HTTPS and WebSockets not currently supported by EthSigner